ace/sources/Springer.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "Springer",
   "identifiers": [
-    "- Springer</title>",
```

Comment:
the tests are failing since the scraper is not recognizing the springer html file, which starts with: `<html><head><title>Combined SpringerLink Page</title>`
How is that title generated? Is that something you do, or is done automatically by beautifulsoup?

Reply:
I tested it with the new Springer identifiers and it seems like it passes the test, but I could future-proof this push by soupifying the Springer HTML to where it would then add a unique identifier (<title>Combined SpringerLink Page</title>) at the top to where Springer articles would be the only ones with the specific title/identifier. However, I do not know to what extent this would slow the scraper down.

Reply:
@aldensyi did you test this in the real world also? 

Do all three identifiers have to be found or just 1 for it to be identified as a SpringLink page?

It looks good to me though if you can confirm it works.

Reply:
@adelavega The article just has to match one of the three, at another point the tests were failing because "Springer" by itself was used as an identifier and causing non-springer articles to be mis-classified as Springer articles.

@aldensyi  Is the test springer HTML included in this pull request relevant? Or has it been manually modified? I'm curious where the top line `<html><head><title>Combined SpringerLink Page</title>` came from. 

> but I could future-proof this push by soupifying the Springer HTML to where it would then add a unique identifier (<title>Combined SpringerLink Page</title>) at the top to where Springer articles would be the only ones with the specific title/identifier. However, I do not know to what extent this would slow the scraper down.

No need to do this AFAICT.

Reply:
I thought the identifier was always a piece of HTML:
```
- Springer</title>
```
So I'm worried if that was failing before will it really work now in the real world? 

The combined line was a suggestion by Alden to inject HTML manually at scraping to identify. I don't think we need to do that though.

Reply:
okay, thanks for the clarification!
I don't think the identifier you pointed out was failing, at some point the identifier was just the string "Springer", so if the word "Springer" was in the article, it was identified as a Springer article.

Looking at the source html, I would guess this line would be consistent across Springer articles:
`<meta name="dc.publisher" content="Springer"/>` as long as they are consistent with their metadata, I'm wary of including strings without HTML tags since that could be mentioned anywhere in the html/text.

Reply:
Yeah I think that would be a better identified @aldensyi can you implement this?

Reply:
I was trying to implement using a whole element but it seems like since the element already contains quotation marks it seems to error out

Reply:
i.e.: `'<meta name="dc.publisher" content="Springer"/>'`

Reply:
I tried formatting it to where I followed that identifier, but it returns an error; So I implemented a few more identifiers to supplement the current list of identifier for Springer.json